Source: @pullmoll
Upstream: no
Reason: Need _GNU_SOURCE defined to define cpu_set_t in sched.h

--- a/src/slurmd/slurmd/req.c	2019-12-20 21:43:45.000000000 +0100
+++ b/src/slurmd/slurmd/req.c	2020-02-23 13:07:29.257513065 +0100
@@ -49,6 +49,7 @@
 #include <numa.h>
 #endif
 #include <poll.h>
+#define _GNU_SOURCE
 #include <pthread.h>
 #include <sched.h>
 #include <signal.h>
